CMakeLists.txt

```diff
@@ -58,12 +58,8 @@ set(NGRAPH_INCLUDE_PATH
 )
 
 if (APPLE)
-    # Suppress an OS X-specific warning.
-    if (POLICY CMP0042)
-        cmake_policy(SET CMP0042 NEW)
-    else()
-        set(MACOS_RPATH ON)
-    endif()
+    # Enable MACOS_RPATH by default.
+    cmake_policy(SET CMP0042 NEW)
```

Comment:
CMake required version is at least 3.1 so no need to check if POLICY CMP0042 is available.